src/test/java/api/requests/InstanceRequestsAPICreationTests.java

```diff
@@ -171,75 +180,292 @@ public void cannotSuccessfullyPlaceATitleLevelRequestWhenNoCopyIsAvailable()
+    placeHoldRequest(item2, pickupServicePointId, patrons2, requestExpirationDate2);
+
+    IndividualResource instanceRequester = usersFixture.charlotte();
+
+    JsonObject requestBody = createInstanceRequestObject(instance.getId(), instanceRequester.getId(),
+      pickupServicePointId, instanceRequestDate, instanceRequestDateRequestExpirationDate);
+
+    CompletableFuture<Response> postCompleted = new CompletableFuture<>();
+
+    client.post(InterfaceUrls.requestsUrl("/instances"), requestBody,
+      ResponseHandler.any(postCompleted));
+
+    Response postResponse = postCompleted.get(10, TimeUnit.SECONDS);
+    assertEquals(201, postResponse.getStatusCode());
+
+    JsonObject representation = postResponse.getJson();
+    //Item2 should have been chosen because it has the nearest requestExpirationDate
+    validateInstanceRequestResponse(representation, pickupServicePointId, instance.getId(), item2.getId(), RequestType.HOLD);
+  }
+
+  @Test
+  public void canPlaceATitleLevelRequestOnUnvailableCopyWhenThereIsAvailableItem()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    //The scenario we're checking is if a user has already made a request on an available iemm, hasn't picked it up, but then
+    //goes back to place an instance level request. The system will find this item and rejects the request for the user, so the user
+    //could be given a successful request on an unavailable item instead, if there is any.
+    UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+
+    DateTime instanceRequestDate = new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC);
+    DateTime instanceRequestDateRequestExpirationDate = instanceRequestDate.plusDays(30);
+
+    LocalDate requestDate = new LocalDate(2017, 7, 22);
+    LocalDate requestExpirationDate1 = requestDate.plusDays(30);
+    LocalDate requestExpirationDate2 = requestDate.minusDays(30);
+
+    IndividualResource instance = instancesFixture.basedUponDunkirk();
+    IndividualResource holdings = holdingsFixture.defaultWithHoldings(instance.getId());
+
+    //create 2 copies with no location id's assigned.
+    final IndividualResource item1 = itemsFixture.basedUponDunkirkWithCustomHoldingAndLocation(holdings.getId(), null);
+    final IndividualResource item2 = itemsFixture.basedUponDunkirkWithCustomHoldingAndLocationAndCheckedOut(holdings.getId(), null);
+
+    //Set up request queues. Item1 has requests (1 queued request), Item2 is requests (1 queued), either could be satisfied.
+    List<IndividualResource> patrons1 = new ArrayList<>();
+    patrons1.add(usersFixture.jessica());
+    placePagedRequest(item1, pickupServicePointId, patrons1, requestExpirationDate1);
+
+    List<IndividualResource> patrons2 = new ArrayList<>();
+    patrons2.add(usersFixture.steve());
+    patrons2.add(usersFixture.rebecca());
+    placeHoldRequest(item2, pickupServicePointId, patrons2, requestExpirationDate2);
```

Comment:
Does this just create two hold requests?

It might be a stylistic difference, can you help me understand how this could be clearer than something like:
```
placePagedRequest(item1, pickupServicePointId, jessica, requestExpirationDate1);
placeHoldRequest(item2, pickupServicePointId, steve, requestExpirationDate2);
placeHoldRequest(item2, pickupServicePointId, rebecca, requestExpirationDate2);
```

Reply:
Yes, it creates two hold requests. This is another way of doing it, and it looks clearer.  I was thinking of loops over readability.